### DISCUSSION TITLE
I just wanted saying thanks for mainting this project

### DISCUSSION THREAD

**Discussion by axkibe** [git-lfs/git-lfs#5628] [General]
Over the last months while I've been embracing LFS with my own implementation and had some questions, feedback and sometimes even a bit critique you guys have nothing but beeing helpful, reasonable and responsive (in either accepting or explaining to me where I'm wrong)

**Comment by bk2204** [comment]
You're very welcome.  Thanks for helping out and being a positive contributor.
